components/NewsAndUpdatesModal.js

```diff
@@ -0,0 +1,181 @@
+              fontStyle="normal"
+              fontWeight="500"
+              lineHeight="36px"
+              color="black.900"
+              margin="0px 16px"
+            >
+              News and Updates
+            </P>
+          </Flex>
+        </Flex>
+      </ModalHeader>
+      <hr />
+      <ModalBody>
+        <Query
+          query={newsAndUpdatesQuery}
+          variables={{ collectiveSlug: 'opencollective', onlyChangelogUpdates: true }}
+          context={API_V2_CONTEXT}
+        >
+          {({ data, loading }) =>
+            loading === false && data ? (
+              <StyledCarousel contentPosition="left">
+                {data.account.updates.nodes.map(update => (
+                  <Container key={update.id}>
+                    <Container
+                      fontFamily="Inter"
+                      fontStyle="normal"
+                      fontWeight="normal"
+                      fontSize="13px"
+                      lineHeight="20px"
+                      color="black.700"
+                    >
+                      <FormattedDate value={update.publishedAt} day="numeric" month="long" year="numeric" />
+                    </Container>
+                    <Flex>
+                      <Span paddingTop="10px">
+                        <NextIllustration
+                          width={12}
+                          height={12}
+                          src="/static/images/news-and-updates-ellipse.svg"
+                          alt="News and Updates Ellipse"
+                        />
+                      </Span>
+                      <P
+                        fontSize="20px"
+                        margin="0px 12px"
+                        fontFamily="Inter"
+                        fontStyle="normal"
+                        fontWeight="500"
+                        lineHeight="36px"
+                        color="black.900"
+                      >
+                        {update.title}
+                      </P>
+                    </Flex>
+                    <Flex pt={2} pb={3}>
+                      <StyledLink
+                        onClick={onClose}
+                        as={Link}
+                        href={`/opencollective/updates/${update.slug}`}
+                        color="blue.700"
+                        fontSize="14px"
+                        display="flex"
+                      >
+                        <FormattedMessage
+                          id="NewsAndUpdates.link.giveFeedback"
+                          defaultMessage="Read more & give Feedback"
+                        />
+                      </StyledLink>
+                    </Flex>
+                    <Flex pb={1}>
+                      <HTMLContent color="black.800" mt={1} fontSize="16px" content={update.summary} />
+                    </Flex>
+                    <Flex pb={3}>
+                      {update.summary.slice(update.summary.length - 3) === '...' && (
```

Comment:
:ice_cream: Seems simpler

```suggestion
                      {update.summary.endsWith('...') && (
```